### PR TITLE
chaosd: add a doc to introduce how to pause the program on service mode

### PR DESCRIPTION
**Pull request by vampireqian**
Signed-off-by: vampireqian <vampire.luo@foxmail.com>

### CODE REVIEW

File: docs/simulate-process-chaos-in-physical-nodes.md [hunk 1]
```diff
@@ -175,6 +175,30 @@ The result is as follows:
 {"status":200,"message":"attack successfully","uid":"c3c519bf-819a-4a7b-97fb-e3d0814481fa"}
 ```
 
+##### killing a process
```
Comment (WangXiangUSTC) [inline docs/simulate-process-chaos-in-physical-nodes.md:178]: could delete L168~L176, it is duplicate with this section

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-process-chaos-in-physical-nodes.md [hunk 2]
```diff
@@ -165,6 +165,8 @@ Attack process python successfully, uid: 9cb6b3be-4f5b-4ecb-ae05-51050fcd0010
 
 #### 使用服务模式模拟进程故障示例
 
+##### 进程被杀死
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-process-chaos-in-physical-nodes.md:168]:
```suggestion
##### 进程被终止
```

File: i18n/zh/docusaurus-plugin-content-docs/version-2.1.5/simulate-process-chaos-in-physical-nodes.md [hunk 3]
```diff
@@ -162,6 +162,8 @@ Attack process python successfully, uid: 9cb6b3be-4f5b-4ecb-ae05-51050fcd0010
 
 #### 使用服务模式模拟进程故障示例
 
+##### 进程被杀死
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.1.5/simulate-process-chaos-in-physical-nodes.md:165]:
```suggestion
##### 进程被终止
```

File: docs/simulate-process-chaos-in-physical-nodes.md [hunk 4]
```diff
@@ -165,16 +165,30 @@ When running an experiment, remember to record the UID of the experiment. When y
 
 #### Examples for simulating process faults using the service mode
 
+##### killing a process
```
Comment (en-jin19) [inline docs/simulate-process-chaos-in-physical-nodes.md:168]:
```suggestion
##### Stop a process
```

File: docs/simulate-process-chaos-in-physical-nodes.md [hunk 5]
```diff
@@ -165,16 +165,30 @@ When running an experiment, remember to record the UID of the experiment. When y
...
 ```
 
-The result is as follows:
+The result is as follows：
 
 ```bash
 {"status":200,"message":"attack successfully","uid":"c3c519bf-819a-4a7b-97fb-e3d0814481fa"}
 ```
 
+##### stopping a process
```
Comment (en-jin19) [inline docs/simulate-process-chaos-in-physical-nodes.md:180]:
```suggestion
##### Pause a process
```

**Comment by en-jin19** [comment]
@vampireqian please update the English document in v2.1.5, thanks!

File: docs/simulate-process-chaos-in-physical-nodes.md (hunk 5 above)
Comment (WangXiangUSTC) [reply docs/simulate-process-chaos-in-physical-nodes.md:180]: The signal `19` corresponds to `SIGSTOP`, I think `Stop`s OK

File: docs/simulate-process-chaos-in-physical-nodes.md (hunk 4 above)
Comment (WangXiangUSTC) [reply docs/simulate-process-chaos-in-physical-nodes.md:168]: The signal `15` corresponds to `SIGTERM`, maybe `terminate` is better

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Comment by vampireqian** [comment]
> @vampireqian请更新v2.1.5中的英文文档，谢谢！

ok

**Comment by vampireqian** [comment]
> @vampireqian please update the English document in v2.1.5, thanks!

ok

File: docs/simulate-process-chaos-in-physical-nodes.md [hunk 6]
```diff
@@ -165,16 +165,30 @@ When running an experiment, remember to record the UID of the experiment. When y
...
 #### Examples for simulating process faults using the service mode
 
+##### Terminate a process
+
 ```bash
 curl -X POST 172.16.112.130:31767/api/attack/process -H "Content-Type:application/json" -d '{"process":"12345","signal":15}'
 ```
 
-The result is as follows:
+The result is as follows：
```
Comment (Oreoxmt) [inline docs/simulate-process-chaos-in-physical-nodes.md:174]:
```suggestion
The result is as follows:
```

File: docs/simulate-process-chaos-in-physical-nodes.md [hunk 7]
```diff
@@ -165,16 +165,30 @@ When running an experiment, remember to record the UID of the experiment. When y
...
 {"status":200,"message":"attack successfully","uid":"c3c519bf-819a-4a7b-97fb-e3d0814481fa"}
 ```
 
+##### Stop a process
+
+```bash
+curl -X POST 172.16.112.130:31767/api/attack/process -H "Content-Type:application/json" -d '{"process":"12345","signal":19}'
+```
+
+The result is as follows：
```
Comment (Oreoxmt) [inline docs/simulate-process-chaos-in-physical-nodes.md:186]:
```suggestion
The result is as follows:
```

File: versioned_docs/version-2.1.5/simulate-process-chaos-in-physical-nodes.md [hunk 8]
```diff
@@ -162,18 +162,32 @@ When running an experiment, remember to record the UID of the experiment. When y
...
 #### Examples for simulating process faults using the service mode
 
+##### Terminate a process
+
 ```bash
 curl -X POST 172.16.112.130:31767/api/attack/process -H "Content-Type:application/json" -d '{"process":"12345","signal":15}'
 ```
 
-The result is as follows:
+The result is as follows：
```
Comment (Oreoxmt) [inline versioned_docs/version-2.1.5/simulate-process-chaos-in-physical-nodes.md:171]:
```suggestion
The result is as follows:
```

File: versioned_docs/version-2.1.5/simulate-process-chaos-in-physical-nodes.md [hunk 9]
```diff
@@ -162,18 +162,32 @@ When running an experiment, remember to record the UID of the experiment. When y
...
 {"status":200,"message":"attack successfully","uid":"c3c519bf-819a-4a7b-97fb-e3d0814481fa"}
 ```
 
+##### Stop a process
+
+```bash
+curl -X POST 172.16.112.130:31767/api/attack/process -H "Content-Type:application/json" -d '{"process":"12345","signal":19}'
+```
+
+The result is as follows：
```
Comment (Oreoxmt) [inline versioned_docs/version-2.1.5/simulate-process-chaos-in-physical-nodes.md:183]:
```suggestion
The result is as follows:
```

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**